benchmarks/benchmark_loop/benchmark_main.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import itertools
+import logging
+from argparse import ArgumentParser
+from tqdm import tqdm
+
+from blackbox_repository.tabulated_benchmark import BlackboxRepositoryBackend
+from benchmarks.benchmark_loop.baselines import methods
+from benchmarks.benchmark_loop.benchmark_definitions import benchmark_definitions
+
+from syne_tune.backend.simulator_backend.simulator_callback import SimulatorCallback
+from syne_tune.stopping_criterion import StoppingCriterion
+from syne_tune.tuner import Tuner
+from coolname import generate_slug
+
+
+if __name__ == '__main__':
```

Comment:
This is leaving a lot of options at the defaults (for example, rung_levels, grace_period, etc, for HyperbandScheduler).

I suppose this is intended, to keep things simple. I see.

Moving forward, and in another PR, we should maybe add several benchmark_main.py examples for the different most important setups we have, and then expose the most important args.

In the end, the use case of running everything on everything is not so useful, most users will rather want to run one method, but then maybe explore its options.

Reply:
Yes, my view is that most of the time to default of an HPO method will not be tuned and we want to evaluate the method with its default.

> In the end, the use case of running everything on everything is not so useful, most users will rather want to run one method, but then maybe explore its options.
Perhaps yes, this is intended to be an example of a simple run everything on everything (although you can also run specific combinations but the idea is to have a simple example).